src/resources/elements/primeDesignSystem/ppopup-modal/ppopup-modal.ts

```diff
@@ -54,4 +54,5 @@ export interface IPopupModalModel {
   buttonTextPrimary?: string;
   buttonTextSecondary?: string;
   confetti?: boolean;
+  className?: string;
```

Comment:
@dkent600 
Added className config to allow adding className to confirmationPopup